src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -69,8 +69,7 @@ define(function (require, exports, module) {
     };
 
     ParameterEditor.prototype._setSpaces = function (parameters) {
-        this._spaces = [];
-        this._spaces.push(parameters.search(/\S/));
+        this._spaces = [parameters.search(/\S/)];
```

Comment:
Hi @Pomax 
Turns out github squashed some of the spaces I left.
The answer for 
"2 3 3" would be [0, 1, 1, 0]
And for 
"<5 spaces>2<3 spaces>3<2 spaces>3" would be [5, 3, 2, 0]
I'll update the above comment as well